server/middleware/validation.js

```diff
@@ -29,6 +33,30 @@ const validators = {
         }
         return null;
     },
+    isValidDay({ label, value, message }) {
+        if (value) {
+            if (getDay(value) > new Date(getYear(value), getMonth(value), 0).getDate() || getDay(value) < 1) {
+                return message || validationErrors.isValidDay(label);
+            }
+        }
+        return null;
+    },
+    isValidMonth({ label, value, message }) {
+        if (value) {
+            if (getMonth(value) < 1 || getMonth(value) > 12) {
+                return message || validationErrors.isValidMonth(label);
+            }
+        }
+        return null;
+    },
+    isYearWithinRange({ label, value, message }) {
+        if (value) {
+            if (getYear(value) > MAX_ALLOWABLE_YEAR || getYear(value) < MIN_ALLOWABLE_YEAR) {
+                return message || validationErrors.isYearWithinRange(label);
```

Comment:
Can you make it return `${label} must be after 1901` and `${label} must be before 2100` as appropriate?

Reply:
Yep change made